### DISCUSSION TITLE
Maybe don't blanket leave the door open for commercialising regardless of artists' release license(s) which also isn't listed

### DISCUSSION THREAD

**Discussion by controlfreakstudio** [funmusicplace/mirlo#592] [Ideas]
from https://mirlo.space/pages/terms

"Mirlo can use the content you’ve submitted. For the purposes of marketing, you grant to us, and others acting on our behalf, the worldwide, non-exclusive, perpetual, irrevocable, royalty-free, sublicensable, transferable right to use, exercise, and commercialize the copyright, publicity, trademark, and database rights with respect to your Content.

When we use the content, we can make changes, like editing or translating it. You grant us the right to edit, modify, reformat, excerpt, delete, or translate any of your Content."

This might be bog standard for the corporate silos but um, might not be quite expected or fished out of the terms by folks attracted to your over arching ethos/goals. Maybe not so scary if any artists are actually seeing this in the terms during early days but if community scale is achieved...

Might you offer a tick box per release along the lines of "I consent to Mirlo or 'others acting on their behalf' being able to edit, derivitive, market and commercialise with this content regardless of my license". That way it's not a blanket agreement by just joining the community. Making it clear to artists that their NC, ND, SA, hard copyright, basic attribution or whatever will be ignored by joining could be a mild surprise.

Or at a rank minimum tighter scope specific uses and terms rather than wildcard to possibly allay some concern?

As you are doing so much different and well it'd be super to maybe see you per case partner with creative members for consentual promotional uses, or an active opt-in rather than a default allowing "commercialised" unkowns by using the service?

**Comment by simonv3** [comment]
Hey @controlfreakstudio ! Thanks for this. I 100% agree. It's come up a couple of times in the [discord](https://discord.com/channels/1070731899317796974/1195301753986502686/1208519240626348132) [as well](https://discord.com/channels/1070731899317796974/1181888862147059773/1182029058863812619).

Granted, reading discord threads isn't the most appealing, so here's a TLDR "we copied these from Ampled & Bandcamp, we want to talk to a lawyer to make these Good Terms, but we're still very much in the "talking to a lawyer to figure out how we decide things" stage as a co-op. We're a very small team of people right now and we have literally no money to pay for the legal aspects of this (nevermind all the stuff we _want_ to do).

So, having said that, I'm wondering about what a way to sooth these (very valid!) concerns are? I bet there's wording that someone could draft that protects us in case we share a screen shot with an artists album art in it, or put a song in one of our blog posts. We could change the terms for now to be less permissive. Would you be down to draft something to replace those paragraphs? Additionally, do you know of any lawyers that might be willing to work on this pro-bono?

**Reply by controlfreakstudio** [reply]
Yes exactly. Cheers for getting it. I want you to succeed and understand resource bottleneck.

Seems there is a massive gap between something covering Mirlo for what would be considered fair and reasonable usage for basic presentation of artist work within the service and what is in the boilerplater as infinite derivitive and commercialisation by Mirlo or others on behalf etc.

Much less permissive with Mirlo's intended and limited scope for basic service sustaining only as a default might be more clear and comfortable. Unfortunately no idea what the lawyer-y way for that is because usually the power imbalance is what is sadly in that pasted block.

Hopeful there is some way to balance covering Mirlo for delivering a scoped base service that might get embedded or linked etc, but be unapologetically pro artist and will not actively sell them out in any of the usual weird ways commercial scale tends to foment.

Also as before, one of the raddest things you can do is prominently present and pass on artists' preferred terms per release which goes a long way to galvinising a trust that it may be shared out there but does not imply or grant Mirlo or anyone else to count that as some commercial product other than delivering the core service. Attribution and engagement with artists for terms of how to use their works outside the expected basic service is awesome. As is a way for Mirlo to respect artists license terms unless you amicably partner with the artists otherwise for marketing or other actively derivitive uses if not inline with artist license.

**Comment by LouisLouiseKay** [comment]
So to be perfectly transparent we've based a lot of our ToS on Ampled and Bandcamp's. It's such a tricky and lawyery area that it's hard to come up with exactly the right wording that both protects us and the artists and since most of our features emulate Bandcamp and Ampled it was the best suited canvas for us to build upon in the future. 

Here I think it's critical to understand that this consent is just for us to be able to run the service at all. If artist don't grant us that right to use their music and not have to pay PROs for every play, there simply isn't a platform to begin with no matter what entity is behind it. I know that on Bandcamp any play is considered a promotional/marketing play and artists accept that in the terms. This is a necessary prerequisite and it is the very reason why Bandcamp is sustainable and why they jump through hoops to prove they're NOT a streaming service but a music shop. If we didn't have that then we'd need to basically lock content behind a paywall (a platform subscription basically) for people to access the music. If we want to let artists make the music available for free, then we need that consent. 

Do you mean that we should make artist tick that box for every release and have it be more proeminent? As in, not just once but everytime they publish an album (and then throw an error message if it isn't ticked)? Would that help with transparency? We have no interest in "hiding" that aspect, as it's crucial for everyone to understand it imo, we could even communicate about that because I think it's kind of an area where people don't really know what happens legally and all the things that make building these platforms so hard to begin with.

(last point : this is how I understand that part of the terms > you submit content (.ie "songs, albums, cover art, album infos") to us > you grant us right to use it (display it on your mirlo page, on the main page, anywhere on the site where albums etc. appears))

**Reply by controlfreakstudio** [reply]
Drummer not a lawyer, but my thought is it'd be much cooler to say something along the lines of Mirlo WON'T actively do that kind of thing in the boilerplae. If it's an escape clause for weird third party agreements, CDNs or web embeds, then scope to that and clarify that is is only to the point of being able to deliver the service as outlined. That is a far cry from infinite derivitves and commercialisation forever of anything uploaded. So there is a gap to fill with some better scope?

I do reckon the most community thing to do would be is have defaulted to NO tick box for can Mirlo use your work for marketing or something very tightly scoped in a terms page at the upload level for granularity if that is what you are actually conveying.  otherwise, if it's just ass covering for core service partnerships, scope to that without the draconian we pwn you forever paste?

**Comment by simonv3** [comment]
This has been changed to this language:

> Mirlo can use the content you’ve submitted for the purposes of marketing Mirlo and your content. This may, but is not limited to, posts on social media and screenshots of our platform in advertisements.

There's a more ideal situation where we add a boolean for this specific clause.
